core/src/main/java/io/aiven/kafka/tieredstorage/RemoteStorageManager.java

```diff
@@ -238,17 +238,20 @@ private void uploadSegmentLog(final RemoteLogSegmentMetadata remoteLogSegmentMet
         throws IOException, StorageBackendException {
         final String fileKey = objectKey.key(remoteLogSegmentMetadata, ObjectKey.Suffix.LOG);
         try (final var sis = transformFinisher.toInputStream()) {
-            uploader.upload(sis, fileKey);
+            final var bytes = uploader.upload(sis, fileKey);
+            metrics.recordObjectUpload(ObjectKey.Suffix.LOG, bytes);
```

Comment:
We need a way to get the aggregated number of bytes. It seems to me, one of:
1. Add a separate aggregated sensor.
2. Make the suffix a tag instead of a part of the sensor name (so we can sum by this tag).
3. Don't bother with separating by tag, the log files will anyway be 99%.

Reply:
We can follow a similar approach as with topic vs topic-partition, and have an aggregated metric on INFO level, and by object type in DEBUG, wdyt?

Reply:
Sure, giving a try now. Pushed latest changes.